config/rbac/role.yaml

```diff
@@ -43,6 +43,16 @@ rules:
   - '*/scale'
   verbs:
   - '*'
+- apiGroups:
+  - admissionregistration.k8s.io
+  resources:
+  - validatingwebhookconfigurations
+  verbs:
+  - get
+  - list
+  - patch
+  - update
+  - watch
```

Comment:
Does the operator still need this? I don't think so, right?

Reply:
yeah, the operator is now who generates the certificates and also patches the k8s manifests with the caBundle (validatingwebhookconfigurations in this PR and in the nexts also apiservices), the admission webhooks doesn't require any extra permission.